cms/djangoapps/contentstore/views/course.py

```diff
@@ -536,41 +535,6 @@ def _get_rerun_link_for_item(course_key):
     return reverse_course_url('course_rerun_handler', course_key)
 
 
-def _deprecated_blocks_info(course_module, deprecated_block_types):
```

Comment:
You'll need to call this function instead: https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/content/block_structure/api.py#L9